src/LVcon501.jl

```diff
@@ -0,0 +1,30 @@
+# Lukšan and Vlček's problem 5.1 in NLS format
+#
+#   Source:
+#   L. Lukšan and J. Vlček
+#   Sparse and Partially Separable Test Problems for Unconstrained and
+#   Equality Constrained Optimization
+#   Technical report 767, 1999
+#
+# A. S. Siqueira, Curitiba/BR, 05/2018.
+using JuMP
+
+export LVcon501
+
+"""Lukšan and Vlček's problem 5.1 in NLS format:
+Chained Rosenbrock function with trigonometric-exponential constraints
+"""
+function LVcon501(n :: Int = 20)
+
+  n < 3 && error(": number of variables must be ≥ 3")
```

Comment:
Instead of an error, I'd prefer adjusting `n` automatically and throw a warning.